src/lib/viewers/BaseViewer.js

```diff
@@ -989,6 +989,7 @@ class BaseViewer extends EventEmitter {
 
         if (this.areNewAnnotationsEnabled() && this.annotationControls) {
             this.annotator.addListener('annotations_create', this.handleAnnotationCreateEvent);
+            this.handleAnnotationsOnLoad();
```

Comment:
Can we guarantee that the annotations will have loaded by this time? Should we do something event based like an `annotations_fetched` event?